src/config.js

```diff
@@ -9,7 +9,12 @@ export const GET_JOBLIST_API = `http://${JYIP}/notifications`;
 export const EMAIL_API = `http://${JYIP}/users/email`;
 export const LOGIN_API = `http://${JYIP}/users/signi`;
 export const SIGNUP_API = `http://${JYIP}/users/signu`;
+
 export const STORAGE_FORMS_API = `http://${BJIP}/resumes`;
 export const STORAGE_FILES_API = `http://${BJIP}/resumes/files`;
 export const STORAGE_LIST_API = `http://${BJIP}/resumes/lists`;
 export const RESUME_API = `http://${BJIP}/resumes`;
+
+export const JOB_DETAILS_API = `http://${JYIP}/notifications`;
+export const JOB_DETAILS_USER_API = `http://${JYIP}/apply?notification`;
```

Comment:
- API 는 엔드포인트까지만 포함합니다.
- 쿼리 스트링 부분은 중간에 자르지 말아주세요! 이렇게 되면 컴포넌트에서 요청을 보내는 url 부분에서 쿼리 스트링 부분이 `=123` 와 같이 나오게 되어, 이것이 의미하는게 무엇인지 파악하기 어렵습니다.
```suggestion
export const JOB_DETAILS_USER_API = `http://${JYIP}/apply`;
```

Reply:
수정하였습니다!!